### DISCUSSION TITLE
freetube

### DISCUSSION THREAD

**Discussion by shadow-absorber** [catppuccin/catppuccin#2632] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

using the built in options for colorschemes for the flavours of catppuccin

### Any additional comments?

seems there is already some catppuccin themes integrated into freetube... so this is more of a request for official information about it on ports or here or somewhere

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
We've moved away from creating those types of repositories where they are implemented somewhere else, outside of our control with no contact from the original maintainer who has implemented it.

Is it possible to have themes controlled outside of freetube or is it only built-in via their source code?

**Reply by shadow-absorber** [reply]
think it is sadly only built in via their source code
